specs/Slider.md

```diff
@@ -310,6 +310,26 @@ The same behavior as above, except that the events should concern
 
 ### Screen reader accessibility
 
+#### `Fluent UI & Fluent UI Northstar comparison`:
```

Comment:
@joschect check these and make sure you understand if we have a problem that would be solved with an input element, or a couple of input elements for range selection.